### ISSUE TITLE
[Word,PowerPoint]: NVDA is not announcing the item position for all the list items in the app.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8286]
###Steps to reproduce:
1. Open a word doc 
2. Navigate to the Insert Tab--> Shape
3. Observe that NVDA is announcing the shape name but there is no info for the shape position (n of m)
4. Same issue is happening everywhere with all list items, NVDA is not announcing the item position.
 
### Expected behavior:
NVDA should announce the list item name as well as position as being announced by narrator.

### Actual behavior:
NVDA is not announcing the list item position across whole App.

### System configuration:
NVDA version: 2018.1.1
App Name: Word, Powerpoint
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
This is working properly in Office 365, but I can reproduce it for example in Word or Powerpoint 2010.
cc: @lukaszgo1

**Comment by Adriani90** [comment]
To be clear, this issue is about item count (e.g. 1 of 5, 2 of 5, etc.) when navigating with up and down arrow keys in lists and combo boxes, this one regarding insert shapes dialog in particular.

cc: @CyrilleB79 could you please test whether this issue occurs in Word or powerpoint 2016? If not, then I suggest to close this issue since all versions before Office 2016 are not supported anymore.
